pkg/pillar/cmd/zedagent/reportinfo.go

```diff
@@ -495,6 +495,11 @@ func PublishDeviceInfoToZedCloud(ctx *zedagentContext) {
 
 	ReportDeviceInfo.Capabilities = getCapabilities(ctx)
 
+	//report on mismatch
```

Comment:
Comment seems misleaderin. Report if there is a local override would be more accurate.

Reply:
Fixed, thank you